cmd/server/config/parse.go

```diff
@@ -25,18 +28,52 @@ func Parse() (*CliConfig, error) {
 
 // parseFromEnv заполняем конфигурацию переменных из окружения
 func parseFromEnv(params *CliConfig) error {
-	envAddr := os.Getenv("ADDRESS")
-	if envAddr != "" {
-		params.Address = envAddr
+	cnf := struct {
+		Address       string `env:"ADDRESS"`
+		LogLevel      string `env:"LOG_LEVEL"`
+		FileStorage   string `env:"FILE_STORAGE"`
+		Restore       bool   `env:"RESTORE"`
+		StoreInterval int    `env:"STORE_INTERVAL" envDefault:"-1"`
+	}{}
+	err := env.Parse(&cnf)
+	// Если ошибка, то считаем, что вывести конфигурацию из окружения не удалось
+	if err != nil {
+		return err
+	}
+	if cnf.Address != "" {
+		params.Address = cnf.Address
+	}
+	if cnf.LogLevel != "" {
+		params.LogLevel = cnf.LogLevel
+	}
+	if cnf.FileStorage != "" {
+		params.FileStorage = cnf.FileStorage
+	}
+	if _, ok := os.LookupEnv("RESTORE"); ok {
+		params.Restore = cnf.Restore
+	}
```

Comment:
Не совсем понятна логика иницализации? Зачем поверять `os.LookupEnv("RESTORE")`, но при этом устанавливать `params.Restore = cnf.Restore`? Не проще ли просто установить значением сразу равным `cnf.Restore`?
 
```suggestion
	params.Restore = cnf.Restore
```

Reply:
Тут я так сделал, чтобы мы могли поменять значение по умолчанию в константе, там же где все остальные значения по умолчанию. Иначе, при изменении значения по умолчанию, если не указывать, RESTORE, то будет записано значение по умолчанию из структуры для парсинга